README.md

```diff
@@ -53,6 +53,7 @@ Training Extensions embed all the metadata required for inference into model fil
   ```
 
 - Build library:
+
```

Comment:
Please revert this change to let pre-commit pass. For me pre-commit is now green locally. Perhaps, re-creating python env can help